modules/ROOT/pages/administration/aliases.adoc

```diff
@@ -283,6 +283,18 @@ The required privileges are described xref::administration/access-control/dbms-a
 
 | database
 | The name of the target database. label:default-output[]
+
+This column is filtered according to the `ACCESS` privileges of the user.
+However, some privileges enable users to see additional databases regardless of their `ACCESS` privileges:
+
+* Users with `CREATE/DROP/ALTER DATABASE` or `SET DATABASE ACCESS` privileges can see all standard databases.
+* Users with `CREATE/DROP COMPOSITE DATABASE` or `COMPOSITE DATABASE MANAGEMENT` privileges can see all composite databases.
+* Users with `DATABASE MANAGEMENT` privilege can see all databases.
+* Users always have visibility of the `system` database.
+
+If a user has not been granted `ACCESS` privilege nor any of the above special cases above to the target database, then it is not visible to the user
```

Comment:
```suggestion
If a user has not been granted any of these privileges to the target database, then it is not visible and this column will be `null`. 
```

Reply:
This changes the meaning somewhat. The above list of privileges are not database specific, they appy to the DBMS. And we need to mention the ACCESS privilege really as that can apply to specific databases and it is not mentioned above.

Reply:
Ok - I will delete the second "above" in "above special cases above" and then commit.